master/internal/api_experiment.go

```diff
@@ -3042,3 +3042,101 @@ func (a *apiServer) DeleteTensorboardFiles(
 
 	return &apiv1.DeleteTensorboardFilesResponse{}, nil
 }
+
+func getPachydermConfig(config map[string]interface{}, experimentID int32) (map[string]interface{}, error) {
+	dataConfig, ok := config["data"].(map[string]interface{})
```

Comment:
drive by comment: we really should just add this to the expconf instead of using the untyped data blob if we can. this look like it is going to be a really bad ux?

Reply:
I agree, this is pulling from the integration work done on the pachyderm side which is putting it's config into the data section of the experiments it's launching over in determined. Ideally, we could do something closer to what's been done for the integration config for master instead of having data be used in this way.

Reply:
OK, makes sense. Glad you agree the data section isn't ideal long term.

Reply:
I'll setup the expconf with an integrations config and reach out to Brody to see if we can have pachyderm store the config there instead